src/ai/backend/manager/api/session.py

```diff
@@ -2036,6 +2037,34 @@ async def match_sessions(request: web.Request, params: Any) -> web.Response:
     )
 
 
+@server_status_required(READ_ALLOWED)
+@admin_required
+async def get_direct_access_host(request: web.Request) -> web.Response:
```

Comment:
Purpose of this route handler is to provide both host and port information. Let's rename the function so that it can reflect the purpose.